### PR TITLE
Added free CodeIgniter course, and organizing list by sub-categories

### PR DESCRIPTION
**Pull request by ShadowDLL**
## What does this PR do?
Add new free course, and sub-categories

### Description 
Added a amazing course of CodeIgniter, a framework PHP

### Why is this valuable (or not)?
Because frameworks promote code productivity

### How do we know it's really free?
It's just free

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-pt_BR.md [hunk 1]
```diff
@@ -204,14 +206,22 @@
...
 * [Introdução ao PHP orientado a objetos](https://www.udemy.com/php-orientado-a-objetos/) - Diego Mariano (Udemy)
 * [Lógica de Programação com PHP](https://www.youtube.com/playlist?list=PLhTDLccA9vgHHwGZArcUqIZ5AUGwrbZ_A) - Curso Zend Framework (YouTube)
 * [Login com validação e flash messages (PHP)](https://www.udemy.com/login-com-validacao-e-flash-messages-php/) - Alexandre Cardoso (Udemy)
 * [PDO para quem não sabe PDO](https://www.udemy.com/pdo-para-quem-nao-sabe-pdo/) - Alexandre Cardoso (Udemy)
 * [PHP 7 do Básico ao Intermediário](https://www.udemy.com/php-do-basico-ao-intermediario/) - Gunnar Correa (Udemy)
 * [PHP para quem entende PHP](https://www.udemy.com/php-para-quem-entende-php/) - Alexandre Cardoso (Udemy)
 
+#### CodeIgniter
+
+* [CodeIgniter para Iniciantes](https://www.cursou.com.br/informatica/codeigniter/) - RBtech (Cursou)
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:218]: It looks like this is just embedding the YouTube videos, so maybe it'd be better to just link to the YouTube playlist directly?

```suggestion
* [Curso de CodeIgniter para iniciantes](https://www.youtube.com/playlist?list=PLInBAd9OZCzz2vtRFDwum0OyUmJg8UqDV) - RBtech
```

File: free-courses-pt_BR.md [hunk 2]
```diff
@@ -26,6 +26,8 @@
 * [Markdown](#markdown)
 * [Node.js](#nodejs)
 * [PHP](#php)
+    * [CodeIgniter](#codeIgniter)
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:29]: Only two spaces are required.

```suggestion
  * [CodeIgniter](#codeIgniter)
```

File: free-courses-pt_BR.md [hunk 3]
```diff
@@ -26,6 +26,8 @@
...
+    * [Laravel](#laravel)
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:30]:
```suggestion
  * [Laravel](#laravel)
```

**Review by SethiPandi — COMMENTED** [review]
Hey, and welcome to free programming books! :wave:
Just left you some comments!

Just want to check also, it looks to me like the Udemy course you've linked is _not_ free, it costs €19.99. This repository is just for free resources. I don't think this belongs in the repository.

**Comment by ShadowDLL** [comment]
Thank you ^^

**Pull request merged**